### ISSUE TITLE
See post you're replying to

### ISSUE THREAD

**Issue by joshsharp** [bellebethcooper/pico#53]
This will be probably be negated by the reply box in conversation view, but if that doesn't happen (or the separate reply view persists as well) I'd like to be able to see the content of the post I'm replying to, I guess above the text box. I use this a lot on other platforms.

**Comment by bellebethcooper** [comment]
Yeah, I think you're right that this is needed if we keep the new post screen for replies.
